server/routes/users.js

```diff
@@ -20,8 +20,9 @@ export default (storage, scriptManager) => {
       },
       payload: {
         email: req.body.email,
+        username: req.body.username,
         connection: req.body.connection,
-        memberships: req.body.memberships,
+        group: req.body.group,
```

Comment:
This should be memberships which is the correct generic term. Groups is too specific.